Add pet-surface-longitudinal (#186)
aramis-lab/clinica: docs/Pipelines/PET_Surface_Longitudinal.md

<!-- markdownlint-disable MD046 -->
# `pet-surface-longitudinal` - Surface-based longitudinal processing of PET images

> This pipeline performs several processing steps for the analysis of PET data on the cortical surface

Reviewer: ```suggestion
This pipeline performs several processing steps for the longitudinal analysis of PET data on the cortical surface
```

[[Marcoux et al., 2018](https://doi.org/10.3389/fninf.2018.00094)]:

> - co-registration of PET and T1-w MRI (T1) images;

Reviewer: ```suggestion
- co-registration of PET and T1-weighted MR images;
```

- intensity normalization;
- partial volume correction;
- robust projection of the PET signal onto the subject’s cortical surface;
- spatial normalization to a template;
- atlas statistics.

This pipeline relies mainly on tools from **[FreeSurfer](https://surfer.nmr.mgh.harvard.edu/)** and
**[PETPVC](https://github.com/UCL/PETPVC)** [[Thomas et al., 2016](https://doi.org/10.1088/0031-9155/61/22/7975)].

> The only difference with the [`pet-surface`](../PET_Surface) pipeline is that the subject’s cortical surface comes from [`t1-freesurfer-longitudinal`](../T1_FreeSurfer_Longitudinal) pipeline and not [`t1-freesurfer`](../T1_FreeSurfer) pipeline.

Reviewer: ```suggestion
The only difference with the [`pet-surface`](../PET_Surface) pipeline is that the subject’s cortical surface is obtained with the [`t1-freesurfer-longitudinal`](../T1_FreeSurfer_Longitudinal) pipeline and not the [`t1-freesurfer`](../T1_FreeSurfer) pipeline.
```

## Prerequisite

You need to have performed the [`t1-freesurfer-longitudinal`](../T1_FreeSurfer_Longitudinal) pipeline on your T1-weighted MR images.

## Dependencies
<!-- If you installed the docker image of Clinica, nothing is required.-->

If you only installed the core of Clinica, this pipeline needs the installation of
**FreeSurfer 6.0**, **SPM12**, **FSL 6.0** and **PETPVC 1.2.4** (which depends on **ITK 4**) on your computer.
You can find how to install these software packages on the [third-party](../../Third-party) page.

## Running the pipeline

The pipeline can be run with the following command line:

```Text
clinica run pet-surface-longitudinal <bids_directory> <caps_directory> <acq_label> <suvr_reference_region> <pvc_psf_tsv>
```

where:

- `bids_directory` is the input folder containing the dataset in a [BIDS](../../BIDS) hierarchy.

> - `caps_directory` acts both as an input folder (where the results of the `t1-freesurfer` pipeline are stored) and as the output folder containing the results in a [CAPS](../../CAPS/Introduction) hierarchy.

Reviewer: ```suggestion
- `caps_directory` acts both as an input folder (where the results of the `t1-freesurfer-longitudinal` pipeline are stored) and as the output folder containing the results in a [CAPS](../../CAPS/Introduction) hierarchy.
```

    `EffectiveResolutionInPlane` is replaced by both `psf_x` and `psf_y`, `EffectiveResolutionAxial` is replaced by `psf_z` and the `acq_label` column has been added.
    Additionally, the SUVR reference region is now a compulsory argument: it will be easier for you to modify Clinica if you want to add a custom reference region ([PET Introduction](../PET_Introduction) page).
    Choose `cerebellumPons` for amyloid tracers or `pons` for FDG to have the previous behavior.

Pipeline options:

- `-np`: This parameter specifies the number of threads to run in parallel.
We recommend using `your_number_of_cpu - 1`.
Please note that PETPVC is extremely demanding in terms of resources and
may cause the pipeline to crash if many subjects happen to be partial volume corrected at the same time
(Error : `Failed to allocate memory for image`).
To mitigate this issue, you can do the following:

    **1)** Use a working directory when you launch Clinica.

    **2)** If the pipeline crashed, just launch again the command (while giving the same working directory).

>     The whole processing will continue where it left (you can reduce the number of threads to run in parallel the second time.)

Reviewer: ```suggestion
    The whole processing will continue where it left (you can reduce the number of threads to run in parallel the second time).
```

!!! note

>     The arguments common to all Clinica pipelines are described in [Interacting with clinica](../../InteractingWithClinica).

Reviewer: ```suggestion
    The arguments common to all Clinica pipelines are described in [Interacting with Clinica](../../InteractingWithClinica).
```

!!! tip
    Do not hesitate to type `clinica run pet-surface-longitudinal --help` to see the full list of parameters.

!!! warning "Case where several longitudinal IDs are present"
    If a subject has more than two longitudinal IDs (e.g. `long-M00M18` and `long-M00M18M36`),
    Clinica is not currently able to deal with this particular case.
    This will be fixed in the future.

## Outputs

Results are stored in the following folder of the

> [CAPS hierarchy](../../CAPS/Specifications/#pet-surface-surface-based-longitudinal-processing-of-pet-images):

Reviewer: The section pet-surface-longitudinal doesn't exist in the CAPS.

`subjects/<participant_id>/<session_id>/pet/<long_id>/surface`

The main output files are (where `*` stands for `<participant_id>_<session_id>_<long_id>`):

- `*_task-rest_acq-<label>_pet_space-<label>_suvr-<label>_pvc-iy_hemi-<label>_fwhm-<value>_projection.mgh`:
PET data that can be mapped onto meshes.
If the `space` is `fsaverage`, it can be mapped either onto the white or pial surface of FsAverage.

> If the `space` is `native`, it can be mapped onto the white or pial surface of the subject’s surface (i.e. `{l|r}h.white`, `{l|r}h.pial` files from the `t1-freesurfer` pipeline).

Reviewer: ```suggestion
If the `space` is `native`, it can be mapped onto the white or pial surface of the subject’s surface (i.e. `{l|r}h.white`, `{l|r}h.pial` files from the `t1-freesurfer-longitudinal` pipeline).
```

- `*_hemi-{left|right}_midcorticalsurface`:
surface at equal distance between the white matter/gray matter interface and the pial surface (one per hemisphere).
- `atlas_statistics/*_task-<label>_acq-<label>_pet_space-<label>_pvc-iy_suvr-<label>_statistics.tsv`:
TSV files summarizing the regional statistics on the labelled atlases (Desikan and Destrieux).

!!! note

>     The full list of output files from the pet-volume pipeline can be found in the [The ClinicA Processed Structure (CAPS) specifications](../../CAPS/Specifications/#pet-surface-surface-based-longitudinal-processing-of-pet-images).

Reviewer: ```suggestion
    The full list of output files from the `pet-surface-longitudinal` pipeline can be found in the [The ClinicA Processed Structure (CAPS) specifications](../../CAPS/Specifications/#pet-surface-surface-based-longitudinal-processing-of-pet-images).
```
